Psr/Log/LoggerAwareInterface.php

```diff
@@ -14,5 +14,5 @@ interface LoggerAwareInterface
      *
      * @return void
      */
-    public function setLogger(LoggerInterface $logger);
+    public function setLogger(LoggerInterface $logger): void;
```

Comment:
shouldn't you also add the return type in LoggerAwareTrait ?

Reply:
Yep, I should.  Done.